findbugs.properties

```diff
@@ -50,16 +50,21 @@ defaults.mavenArtifactId = sonar-findbugs-plugin
 4.2.10.changelogUrl = https://github.com/spotbugs/sonar-findbugs/releases/tag/4.2.10
 4.2.10.downloadUrl = https://repo.maven.apache.org/maven2/com/github/spotbugs/sonar-findbugs-plugin/4.2.10/sonar-findbugs-plugin-4.2.10.jar
 4.3.0.description = Use SpotBugs 4.8.6, sb-contrib 7.6.4, and findsecbugs 1.13.0
-4.3.0.sqs=[10.8,2025.1.*]
-4.3.0.sqcb=[24.12,25.2.*]
+4.3.0.sqs = [10.8,2025.1.*]
+4.3.0.sqcb = [24.12,25.2.*]
 4.3.0.sqVersions = [9.9,10.7]
 4.3.0.date = 2024-10-30
 4.3.0.changelogUrl = https://github.com/spotbugs/sonar-findbugs/releases/tag/4.3.0
 4.3.0.downloadUrl = https://repo.maven.apache.org/maven2/com/github/spotbugs/sonar-findbugs-plugin/4.3.0/sonar-findbugs-plugin-4.3.0.jar
 4.4.0.description = Use SpotBugs 4.9.1, sb-contrib 7.6.9, and findsecbugs 1.13.0
-4.4.0.sqs=[10.8,LATEST]
-4.4.0.sqcb=[24.12,LATEST]
+4.4.0.sqs = [10.8,LATEST]
+4.4.0.sqcb = [24.12,LATEST]
 4.4.0.sqVersions = [9.9,10.7]
 4.4.0.date = 2025-02-14
 4.4.0.changelogUrl = https://github.com/spotbugs/sonar-findbugs/releases/tag/4.4.0
 4.4.0.downloadUrl = https://repo.maven.apache.org/maven2/com/github/spotbugs/sonar-findbugs-plugin/4.4.0/sonar-findbugs-plugin-4.4.0.jar
+4.4.1.description = Use SpotBugs 4.9.2, sb-contrib 7.6.8, and findsecbugs 1.13.0
+4.4.1.sqVersions = [9.9,LATEST]
```

Comment:
Again, you need to update your script to handle the new values. `sqVersions` is deprecated. Please follow the patter next in your previous two versions.